.github/workflows/commit-checks.yml

```diff
@@ -0,0 +1,17 @@
+name: Commit policy
+on: [pull_request]
+
+jobs:
+  commits_check_job:
+    runs-on: ubuntu-latest
+    name: Check messages
+    steps:
+    - name: Get PR Commits
+      id: 'get-pr-commits'
+      uses: tim-actions/get-pr-commits@master
```

Comment:
[consider][minor] Perhaps worth pinning to a specific commit to help avoid breaking changes and/or supply chain attacks.

Reply:
Excellent point, thanks @feltech, pinned [here](https://github.com/TheFoundryVisionmongers/otio-openassetio/pull/2/commits/53d919cffa4dd528a6f134af9ea49963f06026a8).